lib/collection/request.js

```diff
@@ -56,7 +56,8 @@ _.inherit((
              * @type {String}
              * @todo: Clean this up
              */
-            method: ((options && options.method) || DEFAULT_REQ_METHOD).toUpperCase()
+            method: _.has(options, 'method') && !_.isNil(options.method) ?
```

Comment:
one way would be the following, but it would need extra check on `options` being an object.

```
method: (!_.has(options, REQ_METHOD_PROP) || _.isNil(options && options.method)) ?
  DEFAULT_REQ_METHOD || String(options.method).toUpperCase();
```

another variant of above, which may not use lodash and as such compensate:
```
method: (!options || options.hasOwnProperty(REQ_METHOD_PROP) || _.isNil(options.method)) ?
  DEFAULT_REQ_METHOD || String(options.method).toUpperCase();
```

another would be the following, but it would mean restricting to string and that seems okay-ish to me since it is more forgiving around the hasOwnProperty standpoint:
```
method: (options && _.isString(options.method) && options.method) ? 
  options.method.toUpperCase() : DEFAULT_REQ_METHOD);
```